system-parachains/bridge-hubs/bridge-hub-polkadot/src/xcm_config.rs

```diff
@@ -251,8 +255,11 @@ impl xcm_executor::Config for XcmConfig {
 		WaivedLocations,
 		SendXcmFeeToAccount<Self::AssetTransactor, TreasuryAccount>,
 	>;
-	type MessageExporter =
-		(XcmOverBridgeHubKusama, crate::bridge_to_ethereum_config::SnowbridgeExporter);
+	type MessageExporter = (
+		XcmOverBridgeHubKusama,
+		crate::bridge_to_ethereum_config::SnowbridgeExporterV2,
+		crate::bridge_to_ethereum_config::SnowbridgeExporter,
+	);
```

Comment:
let's add a comment that ordering is important:
```suggestion
	type MessageExporter = (
		XcmOverBridgeHubKusama,
		// `SnowbridgeExporterV2` should always be checked before legacy `SnowbridgeExporter`
		crate::bridge_to_ethereum_config::SnowbridgeExporterV2,
		crate::bridge_to_ethereum_config::SnowbridgeExporter,
	);
```